packages/manager/modules/octavia-load-balancer/src/create/controller.js

```diff
@@ -2,8 +2,9 @@ import { PRODUCT_LINK, REGION_AVAILABILITY_LINK } from './constants';
 
 export default class OctaviaLoadBalancerCreateCtrl {
   /* @ngInject */
-  constructor(coreConfig) {
+  constructor(coreConfig, OctaviaLoadBalancerCreateService) {
     this.user = coreConfig.getUser();
+    this.CreateService = OctaviaLoadBalancerCreateService;
```

Comment:
```suggestion
    this.OctaviaLoadBalancerCreateService = OctaviaLoadBalancerCreateService;
```

Reply:
What is the gain of changing this variable name ? As we are already inside octavia load balancer app, I think CreateService is enough to be understandable